__tests__/campaigns.ts

```diff
@@ -357,28 +357,94 @@ describe('query campaignById', () => {
     );
   });
 
-  it("should throw error when user tries to access another user's campaign", async () => {
-    loggedUser = '2'; // User 2 trying to access User 1's campaign
+  it('should allow any user to access any campaign', async () => {
```

Comment:
Should this really be the case, I don't think anonymous users would really need to know what is boosted, if they must know maybe we can give them access on just the part of the fields, eg. name of the admin that boosted, other data they don't need

Reply:
You can use graphorm to limit which properties are available to who, we have helpers to check current user eg. if you want to check if user is owner and so on.

Reply:
@sshanzel where do you check if user owns the campaign, I don't see it in code? Let's say for `campaignById`

Reply:
Gotcha, let's allow only `id` and `user` fields for non owners. That would be enough to show data you need?

Reply:
I'll add type too as it is pretty important in many cases.

Reply:
Basically we just removed `createdAt` and `state` which I think doesn't sound too bad. But if you really think we should hide them, then sure I'll remove them.

Reply:
@capJavert should be updated by now.

Reply:
Perfect 👌 thank you!